src/main/resources/templates/base.twig

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+
+{% block title %}
+<title>Tracker</title>
+{% endblock %}
+
+<link href="/css/bootstrap.min.css" rel="stylesheet"/>
+<link href="/css/mdb.min.css" rel="stylesheet"/>
+
+{% block body %}
+{% endblock %}
+
```

Comment:
So if we have all pages extend this base.twig, all of the resources below will be included automatically on them?

Reply:
Yeah exactly! I'll post some more examples of how this works